embedding/cpp/flutter_app.cc

```diff
@@ -85,6 +92,11 @@ void FlutterApp::OnRegionFormatChanged(app_event_info_h event_info) {
   FlutterDesktopNotifyLocaleChange(handle_);
 }
 
+void FlutterApp::OnAppControlReceived(app_control_h app_control) {
+  TizenLog::Debug("OnAppControlReceived");
+  FlutterDesktopNotifyAppControl(handle_, app_control);
+}
```

Comment:
- Please move these lines between `FlutterApp::OnTerminate` and `FlutterApp::OnLowMemory` to match the declaration order. Please apply the same thing to the C# embedding for consistency.
- We shouldn't need the debug log.
- Also please add<p>
  ```cpp
  assert(IsRunning());
  ```
  for consistency.